src/server/FormData.ts

```diff
@@ -0,0 +1,24 @@
+import BaseFormData from 'form-data';
+
+export default class FormData extends BaseFormData {
+  /*
+    This is a workaround to make node-fetch to work properly
+    with unknown length streams, which was not released yet.
+    https://github.com/node-fetch/node-fetch/pull/707
+   */
```

Comment:
I'd add a TODO here so that when someone, someday pulls up a list of TODOs in this repo they will see this and revert these workarounds once the upstream changes are made.